### PR TITLE
2 packages from sapristi/easy_logging at 0.7

### PR DESCRIPTION
**Pull request by sapristi**
This pull-request concerns:
-`easy_logging.0.7`: Module to log messages. Aimed at being both powerful and easy to use
-`easy_logging_yojson.0.7`: Configuration loader for easy_logging with yojson backend

---
* Homepage: https://sapristi.github.io/easy_logging/
* Source repo: git+https://github.com/sapristi/easy_logging.git
* Bug tracker: https://github.com/sapristi/easy_logging/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/easy_logging_yojson/easy_logging_yojson.0.7/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+   * handlers associated to each logger will format, filter and treat the message independantly.
+   * use the infrastructure with your own handlers with the [MakeLogging] functor.
+   * use tags to add contextual information to log messages"""
+maintainer: "mathiasmillet@gmail.com"
+authors: "Mathias Millet"
+license: "GPL"
+homepage: "https://sapristi.github.io/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+depends: [
+  "ocaml" {>= "4.04.0"}
```
Comment (kit-ty-kate) [inline packages/easy_logging_yojson/easy_logging_yojson.0.7/opam]:
```suggestion
  "ocaml" {>= "4.07.0"}
```
Something is wrong with OCaml < 4.07
```
#=== ERROR while compiling easy_logging.0.7 ===================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.06.1 | pinned(https://github.com/sapristi/easy_logging/archive/v0.7.tar.gz)
# path                 ~/.opam/4.06/.opam-switch/build/easy_logging.0.7
# command              ~/.opam/4.06/bin/dune build -p easy_logging -j 2
# exit-code            1
# env-file             ~/.opam/log/easy_logging-23-8cb2ce.env
# output-file          ~/.opam/log/easy_logging-23-8cb2ce.out
### output ###
#       ocamlc easy_logging/.easy_logging.objs/byte/easy_logging__Make_logging.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamlc.opt -w -g -bin-annot -I easy_logging/.easy_logging.objs/byte -I /home/opam/.opam/4.06/lib/calendar -no-alias-deps -open Easy_logging__ -o easy_logging/.easy_logging.objs/byte/easy_logging__Make_logging.cmo -c -impl easy_logging/make_logging.ml)
# File "easy_logging/make_logging.ml", line 144, characters 26-43:
# Error: Unbound value Infra.SMap.to_seq
```

File: packages/easy_logging/easy_logging.0.7/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+   * handlers associated to each logger will format, filter and treat the message independantly.
+   * use the infrastructure with your own handlers with the [MakeLogging] functor.
+   * use tags to add contextual information to log messages"""
+maintainer: "mathiasmillet@gmail.com"
+authors: "Mathias Millet"
+license: "GPL"
+homepage: "https://sapristi.github.io/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+depends: [
+  "ocaml" {>= "4.04.0"}
```
Comment (kit-ty-kate) [inline packages/easy_logging/easy_logging.0.7/opam]:
```suggestion
  "ocaml" {>= "4.07.0"}
```
Something is wrong with OCaml < 4.07
```
#=== ERROR while compiling easy_logging.0.7 ===================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.06.1 | pinned(https://github.com/sapristi/easy_logging/archive/v0.7.tar.gz)
# path                 ~/.opam/4.06/.opam-switch/build/easy_logging.0.7
# command              ~/.opam/4.06/bin/dune build -p easy_logging -j 2
# exit-code            1
# env-file             ~/.opam/log/easy_logging-23-8cb2ce.env
# output-file          ~/.opam/log/easy_logging-23-8cb2ce.out
### output ###
#       ocamlc easy_logging/.easy_logging.objs/byte/easy_logging__Make_logging.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamlc.opt -w -g -bin-annot -I easy_logging/.easy_logging.objs/byte -I /home/opam/.opam/4.06/lib/calendar -no-alias-deps -open Easy_logging__ -o easy_logging/.easy_logging.objs/byte/easy_logging__Make_logging.cmo -c -impl easy_logging/make_logging.ml)
# File "easy_logging/make_logging.ml", line 144, characters 26-43:
# Error: Unbound value Infra.SMap.to_seq
```

File: packages/easy_logging/easy_logging.0.7/opam [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "dune" {>= "1.8"}
+  "calendar" {>= "2.0"}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "easy_logging/examples"] {with-test}
```
Comment (kit-ty-kate) [inline packages/easy_logging/easy_logging.0.7/opam]:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs "easy_logging/examples"] {with-test}
```

File: packages/easy_logging_yojson/easy_logging_yojson.0.7/opam [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  "dune" {>= "1.8"}
+  "ppx_deriving" {>= "4.0" & < "5.0"}
+  "ppx_deriving_yojson"
+  "easy_logging" {= version}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "conf_loader_modules/yojson"] {with-test}
```
Comment (kit-ty-kate) [inline packages/easy_logging_yojson/easy_logging_yojson.0.7/opam]:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs "conf_loader_modules/yojson"] {with-test}
```

**Comment by sapristi** [comment]
Thanks @kit-ty-kate, I fixed the opam files, and remove the Seq dependency. 
The new pull request is there : https://github.com/ocaml/opam-repository/pull/15243, so I'm closing this one.

**Pull request closed without merging**